<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>983</width>
    <height>596</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Кінотеатр</string>
  </property>
  <property name="windowIcon">
   <iconset resource="movie_image.qrc">
    <normaloff>:/movie_image/logo_cinema.jpg</normaloff>:/movie_image/logo_cinema.jpg</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>-40</x>
      <y>0</y>
      <width>1024</width>
      <height>601</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="page">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>-80</y>
        <width>1024</width>
        <height>682</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="movie_image.qrc">:/movie_image/cinema_room.jpg</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="small_hall">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>0</y>
        <width>431</width>
        <height>181</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgb(255, 255, 255);
color: rgb(255, 0, 0);
border: none;
font: 26pt &quot;Cambria&quot;;


</string>
      </property>
      <property name="text">
       <string>Перегляд</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <property name="styleSheet">
      <string notr="true">background-color:qlineargradient(spread:reflect, x1:0, y1:1, x2:1, y2:0, stop:0.0168539 rgba(0, 0, 0, 255), stop:0.955056 rgba(255, 255, 255, 255));
border: none;
font: 26pt &quot;Cambria&quot;;



</string>
     </property>
     <widget class="QWidget" name="gridLayoutWidget">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>0</y>
        <width>981</width>
        <height>541</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QLabel" name="label_4">
             <property name="font">
              <font>
               <family>Cambria</family>
               <pointsize>26</pointsize>
               <weight>50</weight>
               <italic>false</italic>
               <bold>false</bold>
              </font>
             </property>
             <property name="text">
              <string>Дім Гуччі</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_8">
             <item>
              <widget class="QPushButton" name="info_2">
               <property name="styleSheet">
                <string notr="true">color: #fff; 
text-decoration: none; 
user-select: cursor;
background:#14141a;
outline: none; 
border: 1px solid #e5c182;
border-radius:5px</string>
               </property>
               <property name="text">
                <string>Опис</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="seans_2">
               <property name="styleSheet">
                <string notr="true">color: #fff; 
text-decoration: none; 
user-select: cursor;
background:#14141a;
outline: none; 
border: 1px solid #e5c182;
border-radius:5px</string>
               </property>
               <property name="text">
                <string>Сеанс</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QLabel" name="photo_2">
           <property name="maximumSize">
            <size>
             <width>220</width>
             <height>300</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="scaledContents">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QLabel" name="label_5">
             <property name="font">
              <font>
               <family>Cambria</family>
               <pointsize>26</pointsize>
               <weight>50</weight>
               <italic>false</italic>
               <bold>false</bold>
              </font>
             </property>
             <property name="text">
              <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Ненаситний&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_9">
             <item>
              <widget class="QPushButton" name="info_3">
               <property name="styleSheet">
                <string notr="true">color: #fff; 
text-decoration: none; 
user-select: cursor;
background:#14141a;
outline: none; 
border: 1px solid #e5c182;
border-radius:5px</string>
               </property>
               <property name="text">
                <string>Опис</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="seans_3">
               <property name="styleSheet">
                <string notr="true">color: #fff; 
text-decoration: none; 
user-select: cursor;
background:#14141a;
outline: none; 
border: 1px solid #e5c182;
border-radius:5px</string>
               </property>
               <property name="text">
                <string>Сеанс</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QLabel" name="photo_3">
           <property name="minimumSize">
            <size>
             <width>122</width>
             <height>164</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>220</width>
             <height>300</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="scaledContents">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <widget class="QLabel" name="label_6">
             <property name="font">
              <font>
               <family>Cambria</family>
               <pointsize>26</pointsize>
               <weight>50</weight>
               <italic>false</italic>
               <bold>false</bold>
              </font>
             </property>
             <property name="text">
              <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Вічні&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
             <property name="wordWrap">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_6">
             <item>
              <widget class="QPushButton" name="info_4">
               <property name="styleSheet">
                <string notr="true">color: #fff; 
text-decoration: none; 
user-select: cursor;
background:#14141a;
outline: none; 
border: 1px solid #e5c182;
border-radius:5px</string>
               </property>
               <property name="text">
                <string>Опис</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="seans_4">
               <property name="styleSheet">
                <string notr="true">color: #fff; 
text-decoration: none; 
user-select: cursor;
background:#14141a;
outline: none; 
border: 1px solid #e5c182;
border-radius:5px</string>
               </property>
               <property name="text">
                <string>Сеанс</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QLabel" name="photo_4">
           <property name="maximumSize">
            <size>
             <width>220</width>
             <height>300</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="scaledContents">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="0" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout">
           <property name="sizeConstraint">
            <enum>QLayout::SetDefaultConstraint</enum>
           </property>
           <item>
            <widget class="QLabel" name="label_3">
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>Cambria</family>
               <pointsize>26</pointsize>
               <weight>50</weight>
               <italic>false</italic>
               <bold>false</bold>
              </font>
             </property>
             <property name="layoutDirection">
              <enum>Qt::LeftToRight</enum>
             </property>
             <property name="styleSheet">
              <string notr="true">text-align: center;
</string>
             </property>
             <property name="text">
              <string>Ghost busters</string>
             </property>
             <property name="textFormat">
              <enum>Qt::AutoText</enum>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_7">
             <item>
              <widget class="QPushButton" name="info_1">
               <property name="styleSheet">
                <string notr="true">color: #fff; 
text-decoration: none; 
user-select: cursor;
background:#14141a;
outline: none; 
border: 1px solid #e5c182;
border-radius:5px</string>
               </property>
               <property name="text">
                <string>Опис</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="seans_1">
               <property name="styleSheet">
                <string notr="true">color: #fff; 
text-decoration: none; 
user-select: cursor;
background:#14141a;
outline: none; 
border: 1px solid #e5c182;
border-radius:5px</string>
               </property>
               <property name="text">
                <string>Сеанс</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QLabel" name="photo_1">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximumSize">
            <size>
             <width>220</width>
             <height>300</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="scaledContents">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>880</x>
        <y>550</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: #fff; 
text-decoration: none; 
user-select: cursor;
background:#14141a;
outline: none; 
border: 1px solid #e5c182;
border-radius:5px;
font-size:15px;</string>
      </property>
      <property name="text">
       <string>На головну</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_3">
     <property name="styleSheet">
      <string notr="true">background-color:qlineargradient(spread:reflect, x1:0, y1:1, x2:1, y2:0, stop:0.0168539 rgba(0, 0, 0, 255), stop:0.955056 rgba(255, 255, 255, 255));
border: none;
font: 10pt &quot;Cambria&quot;;



</string>
     </property>
     <widget class="QTextBrowser" name="textBrowser">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>0</y>
        <width>551</width>
        <height>311</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: #fff; 
text-decoration: none; 
user-select: cursor;
background:#14141a;
outline: none; 
border: 1px solid #e5c182;
font-size:20px;</string>
      </property>
     </widget>
     <widget class="QLabel" name="ph">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>0</y>
        <width>441</width>
        <height>601</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>770</x>
        <y>530</y>
        <width>201</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: #fff; 
text-decoration: none; 
user-select: cursor;
background:#14141a;
outline: none; 
border: 1px solid #e5c182;
border-radius:5px;
font-size:20px;</string>
      </property>
      <property name="text">
       <string>До фільмів</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBox">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>450</y>
        <width>201</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: #fff; 
text-decoration: none; 
user-select: cursor;
background:#14141a;
outline: none; 
border: 1px solid #e5c182;
border-radius:5px;
padding: 5px;
font-size:20px;</string>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>5</number>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBox_2">
      <property name="geometry">
       <rect>
        <x>770</x>
        <y>450</y>
        <width>201</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: #fff; 
text-decoration: none; 
user-select: cursor;
background:#14141a;
outline: none; 
border: 1px solid #e5c182;
border-radius:5px;
padding: 5px;
font-size:20px;</string>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>6</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>390</y>
        <width>201</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: #fff; 
text-decoration: none; 
user-select: cursor;
background:#14141a;
outline: none; 
border: 1px solid #e5c182;
border-radius:5px;
font-size:20px;</string>
      </property>
      <property name="text">
       <string>Ряд</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>770</x>
        <y>390</y>
        <width>201</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: #fff; 
text-decoration: none; 
user-select: cursor;
background:#14141a;
outline: none; 
border: 1px solid #e5c182;
border-radius:5px;
font-size:20px;</string>
      </property>
      <property name="text">
       <string>Місце</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_3">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>530</y>
        <width>201</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: #fff; 
text-decoration: none; 
user-select: cursor;
background:#14141a;
outline: none; 
border: 1px solid #e5c182;
border-radius:5px;
font-size:20px;</string>
      </property>
      <property name="text">
       <string>Перевірити</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>320</y>
        <width>441</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: #fff; 
text-decoration: none; 
user-select: cursor;
background:#14141a;
outline: none; 
border: 1px solid #e5c182;
border-radius:5px;
font-size:20px;</string>
      </property>
      <property name="text">
       <string>Статус місця</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <zorder>pushButton_2</zorder>
     <zorder>textBrowser</zorder>
     <zorder>ph</zorder>
     <zorder>spinBox</zorder>
     <zorder>spinBox_2</zorder>
     <zorder>label_7</zorder>
     <zorder>label_8</zorder>
     <zorder>pushButton_3</zorder>
     <zorder>label_9</zorder>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="movie_image.qrc"/>
 </resources>
 <connections/>
</ui>
